frontend_vue/src/components/base/_BaseButton.spec.ts

```diff
@@ -0,0 +1,60 @@
+import { mount, VueWrapper } from '@vue/test-utils'
```

Comment:
more a question about the location. is this how folks do it in the typescript world, have the tests in a .spec file?

Reply:
instead of having everything in a `/tests` folder?

Reply:
I think the .spec is a namig convention coming from the testing libraries/framework. For what I know, it's standardly used to 'specify' to which component the test are bonded.

I'm not a fan of having tests in a separare folder for Base components. 
Ideally, tests file should serve as documentation as well. If you need a Base component, you  can find helpful to see how they behave by checking their unit tests.

In our case we won't have that many base components, so I think it's ok to keep them together in the base folder. 
The alternative, would be to have a separate folder for each component,  which might contain the component itself, tests, types, utils and whatever else.

I would create a test folder for the other components or for some integration tests

Reply:
yeah thats fine